maven-extension/src/main/java/io/opentelemetry/maven/OpenTelemetrySdkService.java

```diff
@@ -89,60 +82,49 @@ public synchronized void dispose() {
   @Override
   public void initialize() throws InitializationException {
     logger.debug("OpenTelemetry: initialize OpenTelemetrySdkService...");
-    if (StringUtils.isBlank(
-        OtelUtils.getSystemPropertyOrEnvironmentVariable(
-            "otel.exporter.otlp.endpoint", "OTEL_EXPORTER_OTLP_ENDPOINT", null))) {
+    AutoConfiguredOpenTelemetrySdkBuilder autoConfiguredSdkBuilder =
+        AutoConfiguredOpenTelemetrySdk.builder();
+
+    // SDK CONFIGURATION PROPERTIES
+    autoConfiguredSdkBuilder.addPropertiesSupplier(
```

Comment:
It'll be cool if we can follow with populating this with XML properties from the POM file :)